lib/IManifest.ts

```diff
@@ -32,12 +32,24 @@ export async function manifestFromResource(testCaseHandlers: {[uri: string]: ITe
         resource.property.specifications.list
           .map((specificationResource: Resource) =>
             ({ [specificationResource.term.value]:
-                manifestFromSpecificationResource(testCaseHandlers, options, specificationResource) }))))) : null,
+                manifestFromSpecificationResource(testCaseHandlers, options, specificationResource, objectLoader) }))))) : null,
     subManifests: await Promise.all<IManifest>([].concat.apply([],
-      resource.properties.include.map((includeList: Resource) => includeList.list
-        .map(manifestFromResource.bind(null, testCaseHandlers, options))))),
+      // This is here because of the way the rdf-star test suite is published
+      // @see https://github.com/rubensworks/rdf-test-suite.js/pull/78/files#r1026326410
+      ((resource.properties.include.length > 0 || !objectLoader)
+        ? resource
+        : (objectLoader.resources?.[resource.value.slice(0, resource.value.lastIndexOf('.')).replace(/\/manifest$/, '#manifest')] ?? resource)
```

Comment:
Could you do a quick manual check to see if this doesn't break anything when running the tests in other places? For example Comunica and https://github.com/rubensworks/jsonld-streaming-parser.js ?

Reply:
Alright, perfect, thanks for checking!

> PS are you aware that the jsonld-streaming-parser is failing 4 spec tests at the moment

Ah, that's new! Probably because of a recent addition in the test suite.
Will probably run into this very soon when implementing JSON-LD-star there :-)